src/QuizController.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from "react";
+import Quiz  from './Quiz.js'
+import Teoria from "teoria";
+
+const POSSIBLE_INTERVALS = [
+    Teoria.interval('m2'),
+    Teoria.interval('M2'),
+    Teoria.interval('m3'),
+    Teoria.interval('M3'),
+    Teoria.interval('P4'),
+    Teoria.interval('d5'),
+    Teoria.interval('P5'),
+    Teoria.interval('m6'),
+    Teoria.interval('M6'),
+    Teoria.interval('m7'),
+    Teoria.interval('M7'),
+    // TODO: include octave
+
+]
+
+function getRandomInt(max) {
+  return Math.floor(Math.random() * Math.floor(max));
+}
```

Comment:
Might move this out into a utils file! I bet it'll be handy more than just here.

Reply:
That's probably the case!  I will probably defer actually making that happen until I want it somewhere else, though.